local.env

```diff
@@ -8,6 +8,6 @@ OPERATOR_KEY_A= 302e020100300506032b65700422042091132178e72057a1d7528025956fe39b
 # ECDSA HEX Encoded Private Key
 HEX_PRIVATE_KEY_A= 0x2e1d968b041d84dd120a5860cee60cd83f9374ef527ca86996317ada3d0d03e7
 HEX_PRIVATE_KEY_B= 0x45a5a7108a18dd5013cf2d5857a28144beadc9c70b3bdbd914e38df4e804b8d8
-RETRY_DELAY=3000 # ms
-MAX_RETRY=20
+RETRY_DELAY=4000 # ms
```

Comment:
Q: why the increase? Are we seeing this to be necessary?

Reply:
It took more than three runs to get the tests to pass, so I tweaked the values to see if it would reduce the number of runs to get all the tests to pass.